Validation/SP_Wood_Cribs/FDS_Input_Files/SP_Test_9.fds

```diff
@@ -7,6 +7,8 @@
 
 &TIME T_END=720. /    
 
+&CLIP MINIMUM_DENSITY=0.05 /  Account for high temps due to charring
```

Comment:
@mcgratta If this is typical for charring, should we just make it default?  I worry about needing to add this to every WUI case.